pipeline-modules/deployment-service/aws/lambda-gateway.yaml

```diff
@@ -0,0 +1,152 @@
+provisioner: aws
+name: lambda-gateway
+version: 1
+revision: 1
+displayName: "AWS Lambda deployment"
+description: "deploy lambda functions with codepipes and serverless"
+target: "deployment-template"
+keywords: ["bash", "linux","functions"]
+author: "CloudCover"
+meta:
+  inputArtifactType:
+    - ContainerImage
+inputs:
+  type: object
+  properties:
+    region:
+      title: Region name
+      description: region name for cluster
+      type: string
+    app_env:
+      title: Application Environment
+      description: Application Related Envs
```

Comment:
I'm assuming this is for environment variables? If so, the description should be more descriptive:
"Application related environment variables"